### ISSUE TITLE
[BUG] - disableAnimation in <User /> Component

### ISSUE THREAD

**Issue by gdschanu-manhdung** [nextui-org/nextui#3521]
### NextUI Version

2.4.1

### Describe the bug

I got bug with this code:
``` javascript
   <User
      name={fullName(post.user.firstName, post.user.lastName)}
      description={postTime(post)}
      avatarProps={{ src: post.user.avtUrl, }}
      className="text-xl font-bold"
    />
  ```
  
  and I got this error in browser console 
  
![image](https://github.com/user-attachments/assets/f4213122-31bc-4d62-996f-710d9a8397c2)

### Steps to Reproduce the Bug or Issue

Just use the User with avatarProps, then I will get the bugs

### Expected behavior

I just want to resolve this bug

### Operating System Version

Ubuntu 22.04

### Browser

Chrome

**Comment by abhinandan-verma** [comment]
@gdschanu-manhdung  there is no  `disableAnimation` prop in the `User`. See if there is such code in you project. 
or
provide a codesandbox or stackblitz to produce the error. It would help debug efficirtly

**Comment by gdschanu-manhdung** [comment]
@abhinandan-verma 
I found this in @nextui-org/user/dist/use-user.d.ts/UseUserProps
![image](https://github.com/user-attachments/assets/717f70fb-1190-4347-99ab-11ab124f18a3)

**Comment by LeakedDave** [comment]
Is this issue still occurring for you if you upgrade to 2.4.6?

**Comment by wingkwong** [comment]
Please use latest version. This has been fixed and I couldn't reproduce in 2.4.6.
